turbo/mixins.py

```diff
@@ -1,53 +1,58 @@
-from asgiref.sync import async_to_sync
-
-from channels.layers import get_channel_layer
+from django.db.models import Model
 
+import turbo
 from turbo import (
-    channel_name_for_instance,
     CREATED,
     UPDATED,
     DELETED,
     REPLACE,
     REMOVE,
-    APPEND,
-)
+    APPEND, )
 
 
 class BroadcastableMixin(object):
+    default_stream_action = APPEND
+    turbo_streams_template = None
+
+    def get_turbo_streams_template(self, target):
+        raise NotImplementedError
+
+    def get_dom_target(self, target):
+        raise NotImplementedError
```

Comment:
```suggestion
        raise NotImplementedError()
```

Need to instantiate it instead of `raise`ing the class, I'm pretty sure

Reply:
Obsolete due to refactoring.